docs/standard/serialization/binary-serialization.md

```diff
@@ -97,41 +175,162 @@ As the nature of binary serialization allows the modification of private members
+- <xref:System.Reflection.TargetParameterCountException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Resources.MissingManifestResourceException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Resources.MissingSatelliteAssemblyException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Runtime.CompilerServices.RuntimeWrappedException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Runtime.InteropServices.COMException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Runtime.InteropServices.ExternalException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Runtime.InteropServices.InvalidComObjectException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Runtime.InteropServices.InvalidOleVariantTypeException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Runtime.InteropServices.MarshalDirectiveException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Runtime.InteropServices.SEHException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Runtime.InteropServices.SafeArrayRankMismatchException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Runtime.InteropServices.SafeArrayTypeMismatchException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Runtime.Serialization.InvalidDataContractException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Runtime.Serialization.SerializationException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
 - <xref:System.SByte?displayProperty=nameWithType>   
+- <xref:System.Security.AccessControl.PrivilegeNotHeldException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Security.Authentication.AuthenticationException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Security.Authentication.InvalidCredentialException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Security.Cryptography.CryptographicException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Security.Cryptography.Xml.CryptoSignedXmlRecursionException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
```

Comment:
The link doesn't resolve, since we haven't reflected on the type. Can you fence it and add a comment, like this:
\`System.Security.Cryptography.Xml.CryptoSignedXmlRecursionException\` \<!--zz \<xref:System.Security.Cryptography.Xml.CryptoSignedXmlRecursionException?displayProperty=fullName -->